processing/src/main/java/io/druid/query/topn/AlphaNumericTopNMetricSpec.java

```diff
@@ -46,7 +46,7 @@ public AlphaNumericTopNMetricSpec(
   @Override
   public Comparator getComparator(List<AggregatorFactory> aggregatorSpecs, List<PostAggregator> postAggregatorSpecs)
   {
-    return new AlphanumComparator();
+    return ComparatorUtils.NATURAL_COMPARATOR;
```

Comment:
i tihnk we can write it if anyone actualyl needs it, so far i dont think alphanumeric has ever been used